packages/engine/src/renderer/functions/canvas.ts

```diff
@@ -0,0 +1,23 @@
+import { isNode, isWebWorker } from "../../common/functions/getEnvironment";
+
+export const configCanvasElement = (canvasId) => {
+  if (!canvasId) return null;
+
+  const canvas = document.getElementById(canvasId) as HTMLCanvasElement;
+  canvas.ondragstart = (e) => {
+    e.preventDefault();
+    return false;
+  }
+
+  return canvas;
+}
+
+export const createCanvas = () => {
+  if(isWebWorker || isNode) {
+    return;
+  }
+  const canvas = document.createElement("canvas");
+  document.body.appendChild(canvas);
+  configCanvasElement(canvas);
```

Comment:
we're passing a canvas element even though the function requires a canvas identifier? please double check this